pages/charts.js

```diff
@@ -0,0 +1,126 @@
+import Navbar from "src/components/navbar";
+import {
+  Heading,
+  Avatar,
+  Box,
+  Center,
+  Image,
+  Flex,
+  Text,
+  Stack,
+  Button,
+  useColorModeValue,
+  Stat,
+  StatLabel,
+  StatNumber,
+  StatHelpText,
+  StatArrow,
+  StatGroup,
+  FormControl,
+  FormLabel,
+  Input,
+  Container,
+} from "@chakra-ui/react";
+import { getRequest, postRequest } from "src/lib/fetch";
+import React, { useState, useEffect } from "react";
+
+export default function Charts() {
+  const [stockPrice, setStockPrice] = useState("Loading");
+  const [key, setKey] = useState("");
+  const getPrice = async (e) => {
+    const response = await getRequest({
+      url: "/api/price?key=c32db451-3f3d-4055-9d0e-c5c5c8ebf96a",
+    });
+    const price = response?.price;
+    setStockPrice(price ? price : "Error");
+  };
+  useEffect(() => {
+    getPrice();
+  });
```

Comment:
* `*}, []); `
may need to add second func param as empty array to denote no dependencies.